specs/config/config-vocab.html

```diff
@@ -16,7 +16,7 @@
       var oasisBase = "https://docs.oasis-open-projects.org/oslc-op/config/v1.0";
       var rev = "01";
       var thisBase = wdBase;
-      if (status != "WD") {
+      if (status != "PSD") {
```

Comment:
should be wd; psd status should be set in respecConfig

Reply:
fixed